stable/gocd/README.md

```diff
@@ -158,6 +159,8 @@ $ kubectl create secret generic gocd-server-ssh \
 | `agent.image.pullPolicy`                  | Image pull policy                                                                                                                                                                | `IfNotPresent`               |
 | `agent.resources`                         | GoCD agent resource requests and limits                                                                                                                                          | `{}`                |
 | `agent.nodeSelector`                      | GoCD agent nodeSelector for pod labels                                                                                                                                           | `{}`                |
+| `agent.affinity`                         | GoCD agent affinity                                                                                            | `{}`                |
+
```

Comment:
One more small change. Can you get rid of this empty line, please? It messes up the formatting of the table. You can see it [here](https://github.com/helm/charts/blob/ba6793d7d3699e194c50618c02b81437fcb4542c/stable/gocd/README.md#gocd-agent) at the end of the "GoCD Agent" table.

Reply:
Unfortunately, I can't make these simple changes, since I don't have access to your branch - and if I fix it in a subsequent PR, I'll need to go through the same process and wait for someone to accept it (and bumping up the version too). :)